pokedex/data/csv/pokemon_dex_numbers.csv

```diff
@@ -2219,41 +3161,73 @@ species_id,pokedex_id,pokedex_number
 422,1,422
 422,5,60
 422,6,60
+422,21,341
+422,25,88
+422E,16,260
+422E,20,75
```

Comment:
These two rows are invalid. `422E` should be `422`.

Reply:
Thanks, somehow missed those.